checks/system/unix.py

```diff
@@ -119,7 +119,7 @@ def check(self, agentConfig):
         io = {}
         try:
             if Platform.is_linux():
-                stdout, _, _ = get_subprocess_output(['iostat', '-d', '1', '2', '-x', '-k'], self.logger)
+                stdout, _, _ = get_subprocess_output(['iostat', '-d', '1', '2', '-x', '-k', '-N'], self.logger)
```

Comment:
Although this would clearly work as advertised according to the manpage, my concern is that applying this by default in the general case may have unwanted side-effects (like context churn in the backend or broken dashboards/monitors due to the change in name to the tags). I am wondering if it would perhaps be more prudent to add this option behind a feature flag such that one could enable this behavior only if desired.

Reply:
We could indeed make this optional. Though, to play the devil's advocate for a moment, if a user has registered device mapper names, they likely desire this behavior as that would require manual symlinking.

How would you like to expose this option?

Reply:
I am not sure how `agentConfig` works, but that seems like the right way of enabling this to be "more optional".

Reply:
Bump @truthbk